### ISSUE TITLE
When using the UpdateAction Operation (PUT) fill the forms with the data provided by the GET Operation

### ISSUE THREAD

**Issue by leroy0211** [lanthaler/HydraConsole#20]
It would be a nice enhancement when you choose to use the UpdateAction Operation (PUT), the forms below will be filled with the data allready provided by the GET Operation. 
This way if you only want to edit a singular field, all the other information will stay the same instead of setting this to `NULL` by the API.

**Comment by lanthaler** [comment]
Yep, that would indeed be nice. Unfortunately, at the moment I don't have the time to implement this myself. Pull requests are more than welcome though
